app/controllers/works_controller.rb

```diff
@@ -138,6 +138,10 @@ def attachment_selected
     @wizard_mode = true
     @work.files_location = params["attachment_type"]
     @work.save!
+
+    # create a directory for the work if the curator will need to move files by hand
+    @work.s3_query_service.create_directory if @work.files_location != "file_upload"
```

Comment:
I'd suggest factoring out this string and referencing it with a variable where it's needed: `work.rb`, `attachment_select.erb`, and here. It would be easy to introduce a typo, and hard to debug.

Reply:
Maybe a ticket, or just leave it.  Not really a part of this change...